applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -17,7 +17,9 @@
 #include "nrf5340_audio_common.h"
 #include "button_handler.h"
 #include "button_assignments.h"
+#if (CONFIG_BT_LL_ACS_NRF53)
```

Comment:
Do we need to remove this? We might add vendor specific calls for SDC, those would probably live in this file as well.

Reply:
This file is not included when building for other controllers